src/utils/logHealthCheck.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * helper function to do log health check
+ */
+const { logInfo, logSystemEventAsync} = require('../operations/common/logging');
+const { AdminLogger } = require('../admin/adminLogger');
+
+module.exports.handleLogHealthCheck = async () => {
+    let healthy = true;
+    try {
+        logInfo('Log HealthCheck', {});
+        const adminLogger = new AdminLogger();
+        await adminLogger.logInfo('Admin Log HealthCheck');
+        await logSystemEventAsync(
+            {
+                event: 'healthCheck',
+                message: 'Performing system healthcheck',
+                args: {}
+            }
+        );
```

Comment:
Instead of sending a new log every time, can we just connect to the access log mongo cluster and read a log. Ref https://github.com/icanbwell/fhir-server/blob/9531f8ec2f41b26d793ea84f70c0654e90a5ef9d/src/config.js#L128 for the mongo config.

Reply:
I would like to keep creating a log entry here, to signify that the full health-check has been run, if that's OK.